gradle.properties

```diff
@@ -1,3 +1,3 @@
-version=5.6.0-NEXT-SNAPSHOT
+version=5.6.1-NEXT-SNAPSHOT
```

Comment:
Our current standard for develop version is <freshest master tag>-NEXT-SNAPSHOT, so you should not bump here